File: app/src/main/java/com/example/keinomoto/automatchingsearchapp/MySQLiteOpenHelper.java
package com.example.keinomoto.automatchingsearchapp;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

public class MySQLiteOpenHelper extends SQLiteOpenHelper {

    static final String DB = "sqlite_sample.db";
    static final int DB_VERSION = 1;
    static final String CREATE_TABLE = "CREATE TABLE keywords_tbl ( " +
            "_id integer PRIMARY KEY AUTOINCREMENT, " +
            "keyword VARCHAR(255) NOT NULL );";
    static final String DROP_TABLE = "DROP TABLE keywords_tbl ;";

    public MySQLiteOpenHelper(Context c) {
        super(c, DB, null, DB_VERSION);
    }

    public void onCreate(SQLiteDatabase db) {
        db.execSQL(CREATE_TABLE);
    }

    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
        db.execSQL(DROP_TABLE);
        onCreate(db);
    }
}

File: app/src/main/java/com/example/keinomoto/automatchingsearchapp/MySQLiteOpenHelper.kt
package com.example.keinomoto.automatchingsearchapp

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper

class MySQLiteOpenHelper(c: Context) : SQLiteOpenHelper(c, DB, null, DB_VERSION) {

    override fun onCreate(db: SQLiteDatabase) {
        db.execSQL(CREATE_TABLE)
    }

    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {
        db.execSQL(DROP_TABLE)
        onCreate(db)
    }

    companion object {

        internal val DB = "sqlite_sample.db"
        internal val DB_VERSION = 1
        internal val CREATE_TABLE = "CREATE TABLE keywords_tbl ( " +
                "_id integer PRIMARY KEY AUTOINCREMENT, " +
                "keyword VARCHAR(255) NOT NULL );"
        internal val DROP_TABLE = "DROP TABLE keywords_tbl ;"
    }
}
